### PR TITLE
[ramda] Fixed propOr

### PR DESCRIPTION
**Pull request by krydima**
Returns the correct type. (null | void | string | number) for the third argument, added support currying, added support placeholder currying.

### CODE REVIEW

**Comment by krydima** [comment]
Merge the pull request [[ramda_v0.x.x] fixed broken ramda tests](https://github.com/flow-typed/flow-typed/pull/3021) for fix the tests issue. Please.

**Comment by AdamWills** [comment]
@krydima - the PR you mentioned has been merged!

**Comment by krydima** [comment]
> @krydima - the PR you mentioned has been merged!

@AdamWills Thank you!

File: definitions/npm/ramda_v0.x.x/flow_v0.82.x-/ramda_v0.x.x.js [hunk 1]
```diff
@@ -1560,19 +1560,90 @@ declare module ramda {
   ): (key: T) => $ElementType<O, T>;
   declare function prop<T: string, O>(key: T, o: O): $ElementType<O, T>;
 
-  declare function propOr<T, V, A: { [k: string]: V }>(
+  // propOr 1 argument => (function (2 arguments) or (function (1 argument) => function (1 argument)))
+  declare function propOr<T, V: string, A: Object, J: (string | number | null | void)>(
```
Comment (AndrewSouthpaw) [inline definitions/npm/ramda_v0.x.x/flow_v0.82.x-/ramda_v0.x.x.js]: Can you add a legend for what T, V, A, and J stand for? It'll make it easier to track.

**Comment by krydima** [comment]
@AndrewSouthpaw Thanks for your comments. I've fixed it.

**Pull request merged**